event_mass_validation/wizard/confirm_attendees.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models, api
+
+
+class ConfirmEventRegistrations(models.TransientModel):
+
+    _name = "event.registration.massvalidation"
+
+    @api.multi
+    def validate_registrations(self):
+        """Validate selected registrations."""
+        context = dict(self.env.context or {})
+        active_ids = context.get('active_ids', []) or []
+
+        registrations = self.env['event.registration'].browse(active_ids)
+
+        registrations.confirm_registration()
+
+        return {'type': 'ir.actions.act_window_close'}
```

Comment:
IMO as it usually happens in Odoo after processing records through a wizard, it would be better to return a list of confirmed registrations.